code/src/entrance.c

```diff
@@ -504,12 +504,11 @@ void Entrance_CheckEpona(void) {
             0x0157, // Hyrule Field -> Lon Lon Ranch
             0x01F9, // Lon Lon Ranch -> Hyrule Field
             0x01FD, // Market Entrance -> Hyrule Field
-            0x00EA, // ZR Front -> Hyrule Field
-            0x0181, // LW Bridge -> Hyrule Field
+            0x0181, // ZR Front -> Hyrule Field
+            0x0185, // LW Bridge -> Hyrule Field
             0x0129, // GV Fortress Side -> Gerudo Fortress
             0x022D, // Gerudo Fortress -> GV Fortress Side
             0x03D0, // GV Carpenter Tent -> GV Fortress Side
-            0x0496, // Gerudo Fortress -> Thieves Hideout
```

Comment:
This was added intentionally so that the glitch could still be triggered on purpose like in the vanilla game, so I think it should stay.

Original PR for reference: #406

Reply:
I guess that's fine at the moment, I was thinking about future proofing in case the hideout entrances get added to ER someday. Maybe it should have some signifier that it needs to be handled if that happens?

Reply:
That works, although it can't use a function result in a static const array so I chose to handle that check separately.